security/advancedtls/advancedtls_test.go

```diff
@@ -594,14 +619,14 @@ func (s) TestClientServerHandshake(t *testing.T) {
 		// server custom check fails
 		{
 			desc:                       "Client sets peer cert, reload root function with verifyFuncGood; Server sets bad custom check; mutualTLS",
```

Comment:
I see you changed this test case and it's not a new one - just curious as to why you changed it?

Reply:
That's a good catch, I was experimenting with some chains and accidentally submitted (it doesn't influence the outputs). Will revert it back